packages/hatchify-core/src/assembler.ts

```diff
@@ -0,0 +1,113 @@
+import { integer } from "./dataTypes"
+import type { PartialAttribute, PartialSchemaV2, SchemaV2 } from "./types"
+
+interface PartialSchemas {
+  [schemaName: string]: PartialSchemaV2
+}
+
+interface Schemas {
+  [schemaName: string]: SchemaV2
+}
+
+export function createIfNotFound(
+  attribute?: PartialAttribute<number>,
+): PartialAttribute<number> {
+  return attribute || integer({ primary: true, autoIncrement: true, min: 1 })
```

Comment:
`return attribute ?? integer({ primary: true, autoIncrement: true, min: 1 })`

suggested change to account for possible 0 for attribute